doc/ref_model/chapters/chapter08.md

```diff
@@ -8,12 +8,12 @@
   * [8.2.1. Characteristics of a Federated Cloud](#8.2.1)
   * [8.2.2. Telco Cloud](#8.2.2)
   * [8.2.3. Telco Operator Platform Conceptual Architecture](#8.2.3)
- * [8.3. Telco Edge Cloud](#8.3)
+* [8.3. Telco Edge Cloud](#8.3)
   * [8.3.1. Telco Edge Cloud Deployment Environment Characteristics](#8.3.1)
   * [8.3.2. Telco Edge Cloud Infrastructure Characteristics](#8.3.2)
   * [8.3.3. Telco Edge Cloud Infrastructure Profiles](#8.3.3)
-  * [8.3.4. Telco Edge Cloud Infrastructure Characteristics](#8.3.4)
-  * [8.3.5. Comparison of Edge terms from various Open Source Efforts](#8.3.5)
+  * [8.3.4. Telco Edge Cloud: Platform Services Deployment](#8.3.4)
```

Comment:
It's nit picking, but it seems inconsistent to use ":" here and nowhere else in this list.

Reply:
@ildikov Thanks - Done